### ISSUE TITLE
Doesn't escape quotes

### ISSUE THREAD

**Issue by garthk** [jclem/logfmt-elixir#13]
We can't safely round-trip through `encode` and `decode` if there are quotes in the values:
```
iex> [x: "1 \"y=2\""  ] |> Logfmt.encode |> Logfmt.decode
%{"x" => "1 ", "y" => 2}
```

**Comment by bettio** [comment]
Same issue here:
```
Logfmt.encode(%{"foo" => "bar \" dddd"}) |> IO.puts

Output is: foo="bar " dddd"

```

**Comment by bettio** [comment]
This should fix the issue: https://github.com/jclem/logfmt-elixir/pull/17

**Comment by bettio** [comment]
@jclem Is this project still actively maintained and developed? This bug makes logfmt-elixir encoder unsuitable in several different scenarios. :(

**Comment by jclem** [comment]
@bettio It is! I just missed this during a bout of notification bankruptcy.

This is fixed in v3.3.1 and released https://hex.pm/packages/logfmt

Sorry for the delay!
